dev-data.py

```diff
@@ -1,31 +1,35 @@
 import json
 import uuid
-from datetime import datetime
+from datetime import datetime as dt
 from random import randint
+import pytz
```

Comment:
I don't want to make it a dependency since dev-data is only used for development and not in prod. I added a note in readme to install it